setup.cfg

```diff
@@ -116,3 +116,76 @@ console_scripts =
     *.pxi
     *.pyx
     build.bat
+
+
+[flake8]
+disable-noqa = True
+max-line-length = 182
```

Comment:
do we need this? Should black take care of long lines?

My undertanding is that we have stopped talking about style as black is taking care of that.

Reply:
black doesn't make changes that can't be verified by checking that the AST didn't change. there's ways to manually split a 1024+ column line to split into 88 cols that black simply won't ever be able to do